src/modules/dashboard/ui/components/dashboard.user-button.tsx

```
@@ -49,7 +49,7 @@ const DashboardUserButton = () => {
   const { data, isPending } = authClient.useSession() as any;
 
   if (isPending || !data.user.name) {
-    return null;
+    return null;z
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)